### PR TITLE
Add `gitee.io`

### PR DESCRIPTION
**Pull request by liminghua999**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

We provide a code hosting and collaborative development platform for developers/enterprises, with more than 10 million developers, more than 25 million hosting projects, bringing together almost all original open source projects in China, and launched the Enterprise Edition in 2016, providing enterprise-level code hosting services, becoming a leading SaaS service provider in the development field.

Organization Website: 
 **https://gitee.com** 

Reason for PSL Inclusion
====

Gitee.io is a domain name used by Gitee, a Chinese-based provider of Git-based collaboration and code hosting services, to provide a service similar to GitHub Pages.Gitee.io allows users to create websites and web pages using HTML, CSS, and JavaScript, and host them on the Gitee platform.As the use of gitee.io domain names becomes more widespread, there is a potential risk of malicious actors using similar domain names to impersonate Gitee or its services.To mitigate this risk and protect users, we recommend adding gitee.io to the Public Suffix List.This will ensure that only Gitee and authorized entities can create subdomains under gitee.io, providing an additional layer of security for users and businesses using Gitee's services.

Number of users this request is being made to serve:
 **More than 10 million** 

DNS Verification via dig
=======
```
dig +short TXT _psl.gitee.io
"https://github.com/publicsuffix/list/pull/1728"
```

Results of Syntax Checker (`make test`)
```
============================================================================
Testsuite summary for libpsl 0.21.2
============================================================================
# TOTAL: 3
# PASS:  3
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in tests
  CC       test-is-public.o
  CC       test-is-public-all.o
  CC       test-is-cookie-domain-acceptable.o
  CC       test-is-public-builtin.o
  CC       test-registrable-domain.o
  CCLD     test-is-cookie-domain-acceptable
  CCLD     test-is-public-builtin
  CCLD     test-is-public
  CCLD     test-is-public-all
  CCLD     test-registrable-domain
PASS: test-is-public-builtin
PASS: test-is-public
PASS: test-is-cookie-domain-acceptable
PASS: test-registrable-domain
PASS: test-is-public-all
============================================================================
Testsuite summary for libpsl 0.21.2
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc
```
=========

### CODE REVIEW

**Review by dnsguru — CHANGES_REQUESTED** [review]
Please remove the extra newline 12196

**Comment by dnsguru** [comment]
> > Please remove the extra newline 12196
> 
> done

Excellent.  And now... patience.  This project is maintained by volunteers and we are just getting to requests from April of last year.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12189,6 +12189,10 @@ githubusercontent.com
 githubpreview.dev
 github.io
 
+// Gitee  : https://www.gitee.io
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12192]: This should be sorted above Github.

File: public_suffix_list.dat [hunk 2]
```diff
@@ -12189,6 +12189,10 @@ githubusercontent.com
...
+// Submitted by LiMingHua <liminghua@oschina.net>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12193]: Please use a non-personal e-mail.

**Review by simon-friedberger — CHANGES_REQUESTED** [review]
* [ ] Expiration (Note: Must STAY above 2y at all times)
  * [ ] gitee.io expires 2025-02-10
* [x] DNS _psl entries
* [x] Tests pass
* [ ] Sorting
* [x] Reasoning/Organization description

**Pull request closed without merging**